### PR TITLE
Add sponsor tile to firmware flashing tab

### PR DESCRIPTION
**Pull request by blckmn**
Still need to finalise the Html:
1. To centre the images,
2. Add a title "our sponsors" or similar.

Note the above is independent of this PR (as it is loaded into the API).

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: src/js/tabs/firmware_flasher.js
Comment (haslinghuis) [inline src/js/tabs/firmware_flasher.js:269]: Maybe check for `navigator.onLine`

**Comment by sugaarK** [comment]
I can see the window for the sponser bar.. what do we need to do to get it working @blckmn

**Comment by blckmn** [comment]
> I can see the window for the sponser bar.. what do we need to do to get it working @blckmn 

Some html tidy up :) possibly detection between dark and light mode... not sure if we should do that in the app or API...

**Review by haslinghuis — APPROVED** [review]

**Review by sugaarK — APPROVED** [review]

**Comment by TheIsotopes** [comment]
@blckmn @haslinghuis I don't think this was intentional. Can someone please change this.

Screenshot 2023-06-06 095612

**Comment by haslinghuis** [comment]
Oops this was not intentional.

**Comment by TheIsotopes** [comment]
working now ... thx @haslinghuis

**Pull request merged**
